### PR TITLE
Missing iostream include

### PR DESCRIPTION
**Pull request by ananace**

### CODE REVIEW

File: apps/openmw/mwworld/contentloader.hpp [hunk 1]
```diff
@@ -2,6 +2,7 @@
 #define CONTENTLOADER_HPP
 
 #include <iosfwd>
+#include <iostream>
```
Comment (lgromanowski) [inline apps/openmw/mwworld/contentloader.hpp:5]: Hmm, is iostream include really needed here? There is iosfwd included already.

**Comment by zinnschlag** [comment]
Yes, it is. Because of this line:

> std::cout << "Loading content file " << filepath.string() << std::endl;

**Comment by zinnschlag** [comment]
To elaborate a bit more on that: iosfwd gives you forward declarations of the stream types. It does not give you the cout object and it does not give you the overloaded << operator.

**Comment by lgromanowski** [comment]
Ok, you're right.

**Pull request merged**